src/Core/Framework/Gateway/Context/Command/Handler/AbstractContextGatewayCommandHandler.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Gateway\Context\Command\Handler;
+
+use Shopware\Core\Framework\Gateway\Context\Command\AbstractContextGatewayCommand;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+/**
+ * @template TCommand of AbstractContextGatewayCommand
```

Comment:
You can set a default and shorten all `AbstractContextGatewayCommandHandler<AbstractContextGatewayCommand>` instances (e.g. in the ContextGatewayCommandRegistry):
```suggestion
 * @template TCommand of AbstractContextGatewayCommand = AbstractContextGatewayCommand
```